actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorProfiler.java

```diff
@@ -44,7 +43,7 @@
 {
     private final HashMap<Object, ProfilerData> collectedData = new HashMap<>();
     private final ReadWriteLock rwlock = new ReentrantReadWriteLock();
-    private final KryoCloner kryoCloner = new KryoCloner();
+    private final KryoSerializer kryoCloner = new KryoSerializer();
```

Comment:
Oh, my apologies. This is only used in the Orbit benchmarks so we do still need it, but otherwise it's not used at runtime in a real application.